en/download.rst

```diff
@@ -6,33 +6,31 @@
 Download
 ================================================================================
 
-The latest stable release of |osgeo-live-version| :
+The latest stable release is |osgeo-live-version| and can be downloaded from:
+https://sourceforge.net/projects/osgeo-live/files/
 
-.. image:: ../images/download_buttons/download-dvd.png
```

Comment:
The current images includes text which doesn't align with the images we offer. If we want to keep images (which I'm ok with), someone will need to update images.